packages/mui-base/src/useAutocomplete/useAutocomplete.js

```diff
@@ -1020,9 +1020,20 @@ export default function useAutocomplete(props) {
     }
 
     firstFocus.current = false;
-  };
 
-  const handleInputMouseDown = (event) => {
```

Comment:
@taverasady Sorry I kind of missed the topic of this PR – I think it would be better to explicitly check [`event.button`](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button) or [`event.buttons` ](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons) to detect a right click

Reply:
@mj12albert  I agree, re-verted changes and added the extra code on the if

Reply:
To check what should be the correct behavior with the middle click

Reply:
The native `<select>` does not open on the middle click (on Windows, at least).